.changeset/quick-cheetahs-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-cloudflare": patch
+---
+
+Skip empty next steps in the output summary. Previously, they would log undefined undefined.
```

Comment:
given the PR's change I would update the changeset 🙏 

something along these lines would be great 🙂
```suggestion
fix: make sure that using the current directory doesn't generate an `undefined undefined` log in the output summary.
```

the `fix:` prefix is as per our [changesets convention](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changeset-message-format)